src/isql/FrontendLexer.cpp

```diff
@@ -28,22 +28,25 @@
 
 
 static std::string trim(std::string_view str);
-
+static inline int fb_isspace(const char c)
+{
+	return isspace((int)(UCHAR)c);
+}
 
```

Comment:
We already have one in isql.epp. (And a lot of places in the code where parameter is casted to UCHAR.)
Why duplicate code? Even if it's trivial.

May be better add such function to fb_utils namespace?

Reply:
It would require a common header file. It looked like Adriano was trying to avoid unnecessary dependencies and keep minimal include list.

Reply:
> I see no problem with having files from src/common anywhere in the project.

Agreed.